.expeditor/scripts/release_habitat/package_and_upload_arm_binary.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -euo pipefail
+
+source .expeditor/scripts/release_habitat/shared.sh
+
+docker run \
+    -it \
+    --rm \
+    --init \
+    --volume "$(pwd)":/workdir \
+    --workdir /workdir \
+    rust:buster \
```

Comment:
Will do that. I initially just left it as is, because the cargo build will anyway cause the installation of the correct toolchain before running any tests.